ultradns/config.go

```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
 	"log"
 
-	"github.com/terra-farm/udnssdk"
+	"github.com/aliasgharmhowwala/ultradns-sdk-go"
```

Comment:
Please remove personal for reference from everywhere.

Reply:
yup will do it once we merge the sdk plugin to master in ultradns project